src/webview/mod.rs

```diff
@@ -4,6 +4,7 @@
 
 //! [`WebView`] struct and associated types.
 
+pub mod proxy;
```

Comment:
lets hide the module and 
```suggestion
mod proxy;
```
and then export the needed types only 
```
pub use proxy::ProxyConfig;
```